File: ueueo-framework/ueueo-domain-driven-desgin/src/main/java/com/ueueo/ddd/domain/entities/events/Distributed/EntityEto.cs
using System;

namespace Volo.Abp.Domain.Entities.Events.Distributed;

[Serializable]
public class EntityEto : EtoBase
{
    public String EntityType;// { get; set; }

    public String KeysAsString;// { get; set; }

    public EntityEto()
    {

    }

    public EntityEto(String entityType, String keysAsString)
    {
        EntityType = entityType;
        KeysAsString = keysAsString;
    }
}

File: abp-frameworks/ueueo-domain-driven-design/src/main/java/com/ueueo/ddd/domain/entities/events/distributed/EntityEto.java
package com.ueueo.ddd.domain.entities.events.distributed;

import com.ueueo.domain.entities.events.distributed.EtoBase;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@EqualsAndHashCode(callSuper = true)
@Data
@AllArgsConstructor
@NoArgsConstructor
public class EntityEto extends EtoBase {
    private String entityType;

    private String keysAsString;

}
